code-challenges.test.js

```diff
@@ -15,19 +20,56 @@
 
 // a) Create a test with an expect statement using the variable provided. HINT: Check out this resource: https://jestjs.io/docs/expect#expectarraycontainingarray
 
-const colors1 = ["purple", "blue", "green", "yellow", "pink"]
-// Expected output example (can be a different order): ["yellow", "blue", "pink", "green"]
-const colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
-// Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
-
-
-// b) Create the function that makes the test pass.
-
+// --->TEST 
+ def("original")("original1"), () => {
+      it("takes in an array, removes the first item from the array and shuffles the remaining content."), () => {
+          expect(original("colors1").toEqual(["yellow", "blue", "pink", "green"]))
+      }
+  }
+
+ const colors1 = ["purple", "blue", "green", "yellow", "pink"]
+//  Expected output example (can be a different order): ["yellow", "blue", "pink", "green"]
+ const colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
+//  Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
+
+
+//  b) Create the function that makes the test pass.
+
+// -->Psuedo
+// Create two variables (Original, original1) attached too the two const colors1 and colors2.
+// Create a third variable (copy) that ties the first two variables and merges them together using concat.
+// Sort the result using .sort and randomize the resutls using Math.random 
+// console log the third variable (copy)
+
+  var original = colors2;
+  var original1 = colors1;
+  var copy = [].concat(original, original1);
+  copy.sort(function(){
+    return 0.5 - Math.random();
+  });
+  console.log(copy);
```

Comment:
In your test it looks like you were calling your function "original" however what is defined here as original is a variable.  It also seems like you might be solving a different problem then the prompt is asking.  Make sure to look at what the expected output is for the test variables and that it matches the direction you are taking.